### ISSUE TITLE
conv2d hyperparameter experiments

### ISSUE THREAD

**Issue by RowanA2002** [RowanA2002/178-convy-net#4]
Experiment with tuning the dilation, padding, and stride and record results. This may make the accuracy worse due to the images being low resolution already but will lead to a better understanding of the convolutional neural network.
